.promu.yml

```diff
@@ -0,0 +1,41 @@
+repository:
+    path: github.com/cloudflare/unsee
+build:
+    binaries:
+        - name: unsee
+          path: .
+    flags: -a -tags netgo
+    ldflags: |
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}
```

Comment:
I don't understand `{{repoPath}}/vendor/github.com/prometheus/common/version.` part here, why is it needed?

Reply:
As you can see on https://github.com/cloudflare/unsee/blob/master/Makefile#L55 there's only one linker flag passed currently